recipes/imutils/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+mesa-libGL
```

Comment:
You might need to use `mesa-libGL-devel` if you need the headers

Reply:
But it looks like you don't use the headers, per se

Reply:
Opencv [doesn't](https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/yum_requirements.txt) use them either. Frankly, all I tried to do was to get a passing build. 😅

Reply:
Which it seems like you have done!

Reply:
Let us know when this is ready for review